docs/modules/reference/pages/daemons/daemon-config-files/trapd.adoc

```diff
@@ -3,6 +3,19 @@
 = Trapd
 
 Handles processing of xref:operation:events/sources/snmp-traps.adoc[SNMP traps] into events.
+Based around traditional UDP-based SNMP traps, it receives traps that remote devices create.
+It enables OpenNMS to receive SNMP traps and convert them into events, and convert those events into alarms and notifications.
+Its functionality is similar to how xref:reference:daemons/daemon-config-files/syslogd.adoc[syslogd] handles syslog entries.
+
+Trapd supports V1 traps, V2 traps and notifications, and V3 traps and notifications.
+It also accepts traps with any community name (V1, V2c).
+You can use that community name in event definitions as part of a mask.
+
+== How it works
+
+When an SNMP trap arrives, trapd parses the message, comparing the sender to known nodes.
+If the sender is an unknown node and the `new-suspect-on-trap` parameter is set to "true", {page-component-title} sends an event to discover the node.
+Otherwise, {page-component-title} tags the trap to the correct node, broadcasts it to eventd, and the trap becomes a {page-component-title} event.
```

Comment:
```suggestion
Otherwise, {page-component-title} tags the trap to the monitored node, broadcasts it to eventd, and the trap becomes a {page-component-title} event.
```